fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedConnector.java

```diff
@@ -56,18 +60,24 @@ public UnifiedConnector(ConnectorContext context) {
         derivedProperties.putAll(context.getProperties());
         derivedProperties.put(HIVE_METASTORE_TYPE, metastoreType);
         derivedProperties.put(ICEBERG_CATALOG_TYPE, metastoreType);
+        derivedProperties.put(PAIMON_CATALOG_TYPE, metastoreType);
         derivedProperties.put(KUDU_CATALOG_TYPE, metastoreType);
 
         ConnectorContext derivedContext = new ConnectorContext(context.getCatalogName(), context.getType(),
                 derivedProperties.build());
 
-        connectorMap = ImmutableMap.of(
-                HIVE, new HiveConnector(derivedContext),
-                ICEBERG, new IcebergConnector(derivedContext),
-                HUDI, new HudiConnector(derivedContext),
-                DELTALAKE, new DeltaLakeConnector(derivedContext),
-                KUDU, new KuduConnector(derivedContext)
-        );
+        boolean shouldCreatePaimonConnector = null != context.getProperties().get(PAIMON_CATALOG_WAREHOUSE);
```

Comment:
why you check this config?

Reply:
Paimon has multiple checks in its SDK. In the real usage of our customers, it may come to many validation problems when getting metadata in unified catalog even if there is no Paimon table in the catalog. So I add a judgement to filter out Paimon if its warehouse configure does not exist.

Reply:
if this is necessary, we can move this line to a place just before line 77.

Reply:
and `contains` would be better.